### PR TITLE
Added Azure Batch pools configuration file for rnaseq pipeline

### PR DESCRIPTION
**Pull request by olesya13**
**Name:** Azure Batch config for nf-core/rnaseq pipeline
**About:** defines several Azure Batch pools with virtual machines from Edv4-series and assigns pools to labels according to the requirements defined in pipeline's base.config

### CODE REVIEW

**Review by abhi18av — APPROVED** [review]
Technilcally, looks good to merge ✅ 

But I'm not sure if this would solve the issue of automated testing on the nf-core side or would facilitate beginner users as new accounts don't have access to `EDv4` VMs 

image

File: conf/pipeline/rnaseq/azurebatch_pools_Edv4.config [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+params {
+    config_profile_description = 'Azure Batch Edv4 pools profile'
+    config_profile_contact = 'Olesya Melnichenko (@olesya13) & Venkat Malladi (@vsmalladi)'
+    config_profile_url = 'https://azure.microsoft.com/services/batch/'
+}
+
+process {
+    queue = 'Standard_E2d_v4'
+    withLabel:process_low   {queue = 'Standard_E2d_v4'}
```
Comment (Emiller88) [inline conf/pipeline/rnaseq/azurebatch_pools_Edv4.config:24]:
```suggestion
    withLabel:process_low {queue = 'Standard_E2d_v4'}
```
Might pick a lane on the alignment or no alignment here.

**Review by Emiller88 — APPROVED** [review]
Looks good to me!

**Comment by maxulysse** [comment]
Isn't this something more for azure at a global level than just for the rnaseq pipeline?

**Comment by vsmalladi** [comment]
> Isn't this something more for azure at a global level than just for the rnaseq pipeline?

Yes that is correct. The quota is at the global level.

**Comment by olesya13** [comment]
@abhi18av Do you think it will be useful to add configuration file for Ddv4 series? We ran experiments on both Ddv4 and Edv4, so I have Ddv4 config ready. Edv4 turned out to be more cost efficient in our experiments, that's the main reason why I decided to add this one.

**Comment by maxulysse** [comment]
Can you add it in the main README too please:
https://github.com/nf-core/configs/blob/master/README.md#pipeline-specific-institutional-documentation

File: conf/pipeline/rnaseq/azurebatch_pools_Edv4.config (hunk 1 above)
Comment (olesya13) [reply conf/pipeline/rnaseq/azurebatch_pools_Edv4.config]: fixed - removed extra spaces

**Comment by olesya13** [comment]
@maxulysse documentation added

**Comment by abhi18av** [comment]
Whoops, dropped the ball on this one - @olesya13 , I'd say that this config is enough for the initial work since on the nf-core Azure account we are capped on 200 low-priority CPUs therefore it should work with that tweak.

Though, later on we'd ideally also test with the `Ddv4` VMs since that particular series has been recommended in the past - https://github.com/seqeralabs/nf-core-tower-azure-poc

**Pull request merged**